src/selector/SearchSelectorManager.ts

```diff
@@ -3,8 +3,7 @@ import { searchTracks } from "cody-music";
 import { MusicDataManager } from "../music/MusicDataManager";
 import { requiresSpotifyAccess } from "../music/MusicUtil";
 
-// lkjlkjlkjlkjlkjlkjljk lksjdf lkj lkj lkj lkj lkj
```

Comment:
lol - I wonder what this comment was for

Reply:
:) event testing